portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -112,15 +124,72 @@ function insertSearch() {
 
   const searchInput = document.createElement('input');
   searchInput.className = 'form-control form-inline';
+  searchInput.id = 'userSearch';
   searchInput.type = 'text';
   searchInput.placeholder = 'Search';
 
   const searchI = document.createElement('i');
   searchI.className = 'fas fa-search text-white ml-3 mr-auto';
 
+  const searchResults = document.createElement('ul');
+  searchResults.id = 'searchResults';
+
   searchDiv.appendChild(searchInput);
   searchDiv.appendChild(searchI);
   searchBar.appendChild(searchDiv);
+  searchBar.append(searchResults);
+
+  return searchBar;
+}
+
+// searches each child Node of the page in the elementArray and retains the
+// elements that contain the wanted word
+function searchPages(elementArray) {
+  const wantedWords = document.getElementById('userSearch').value.toLowerCase();
```

Comment:
@kennzoid 
https://github.com/kofimeighan/adelardos-capstone-2020/pull/17#discussion_r450639680
nit: This is typically called something like searchQuery or query or queryString. I personally wouldn't use wantedWords because it makes me think it's an array of words.